editor/src/scene_viewer/gizmo.rs

```diff
@@ -297,6 +302,11 @@ impl SceneGizmo {
                 },
             )
         }
+
+        if let Some(camera_controller) = camera_controller {
+            camera_controller.yaw = pos.x * 0.1;
```

Comment:
You need to use relative values, not absolute position here. Otherwise the camera rotation will jump unpredictably when you start rotating the camera. So basically you need to save click position somewhere and calculate `let delta = pos - self.click_position` and use `delta` instead of `pos` in your code.

Reply:
Okay interesting, I was using delta originally but when doing some reformatting I noticed it seemed pretty smooth just going off mouse movement. Didn't notice I could get the click position 👍